README.md

```diff
@@ -12,25 +12,30 @@ For an example of additional HTTP filters, see [here](http-filter-example).
 
 To build the Envoy static binary:
 
-1. `git submodule update --init`
-2. `bazel build //:envoy`
+```
+$ bazel build //source:envoy
+```
 
 ## Testing
 
-To run the `echo2` integration test:
+To run the integration tests:
 
-`bazel test //:echo2_integration_test`
+```
+$ bazel test //test/...
+```
 
 To run the regular Envoy tests from this project:
 
-`bazel test @envoy//test/...`
+```
+$ bazel test @envoy//test/...
+```
 
 ## How it works
 
-The [Envoy repository](https://github.com/envoyproxy/envoy/) is provided as a submodule.
+The [Envoy repository](https://github.com/envoyproxy/envoy/) will be downloaded as a .zip via bazel's `http_archive`.
```

Comment:
Yeah, this is an interesting change. Originally we used submodules to mimic what Lyft do when extending Envoy, but I think what you have here is the Bazel-first approach, so seems reasonable.